diff --git a/src/big/mpp/userapps/sample/fastboot_app/main.cc b/src/big/mpp/userapps/sample/fastboot_app/main.cc
--- a/src/big/mpp/userapps/sample/fastboot_app/main.cc
+++ b/src/big/mpp/userapps/sample/fastboot_app/main.cc
@@ -451,8 +451,10 @@ int sample_vivcap_init( void )
 
 #if defined(CONFIG_BOARD_K230_CANMV)
     sensor_type = OV_OV5647_MIPI_CSI0_1920X1080_30FPS_10BIT_LINEAR;
-#elif defined(CONFIG_BOARD_K230_CANMV_V2) || defined(CONFIG_BOARD_K230_CANMV_01STUDIO) || defined(CONFIG_BOARD_K230_CANMV_DONGSHANPI)
+#elif defined(CONFIG_BOARD_K230_CANMV_V2) || defined(CONFIG_BOARD_K230_CANMV_01STUDIO)
     sensor_type = OV_OV5647_MIPI_CSI2_1920X1080_30FPS_10BIT_LINEAR_V2;
+#elif defined(CONFIG_BOARD_K230_CANMV_DONGSHANPI)
+    sensor_type = GC2093_MIPI_CSI2_1920X1080_30FPS_10BIT_LINEAR;
 #elif defined(CONFIG_BOARD_K230D_CANMV) || defined(CONFIG_BOARD_K230D_CANMV_BPI)
     sensor_type = OV_OV5647_MIPI_1920X1080_30FPS_10BIT_LINEAR;
 #elif defined(CONFIG_BOARD_K230_CANMV_LCKFB)
