diff --git a/src/big/mpp/userapps/sample/fastboot_app/main.cc b/src/big/mpp/userapps/sample/fastboot_app/main.cc
--- a/src/big/mpp/userapps/sample/fastboot_app/main.cc
+++ b/src/big/mpp/userapps/sample/fastboot_app/main.cc
@@ -50,13 +50,25 @@
-#if defined(CONFIG_BOARD_K230_CANMV) || defined(CONFIG_BOARD_K230_CANMV_V2) || defined(CONFIG_BOARD_K230_CANMV_01STUDIO)
+#if defined(CONFIG_BOARD_K230_CANMV)
 #define ISP_CHN1_HEIGHT (720)
 #define ISP_CHN1_WIDTH  (1280)
+#define ISP_CHN0_WIDTH  (960)
+#define ISP_CHN0_HEIGHT (540)
+
+#define ISP_INPUT_WIDTH (1920)
+#define ISP_INPUT_HEIGHT (1080)
+
+#define LCD_WIDTH       (540)
+#define LCD_HEIGHT      (960)
+
+#elif defined(CONFIG_BOARD_K230_CANMV_V2) || defined(CONFIG_BOARD_K230_CANMV_01STUDIO)
+#define ISP_CHN1_HEIGHT (720)
+#define ISP_CHN1_WIDTH  (1280)
 #define ISP_CHN0_WIDTH  (1920)
 #define ISP_CHN0_HEIGHT (1080)
 
 #define ISP_INPUT_WIDTH (1920)
 #define ISP_INPUT_HEIGHT (1080)
 
 #define LCD_WIDTH       (1080)
 #define LCD_HEIGHT      (1920)
 
 #elif defined(CONFIG_BOARD_K230D_CANMV) || defined(CONFIG_BOARD_K230_CANMV_DONGSHANPI) || defined(CONFIG_BOARD_K230D_CANMV_BPI) || defined(CONFIG_BOARD_K230_CANMV_LCKFB)
@@ -291,8 +303,10 @@ k_s32 sample_connector_init(void)
 {
     k_u32 ret = 0;
     k_s32 connector_fd;
-#if defined(CONFIG_BOARD_K230_CANMV) || defined(CONFIG_BOARD_K230_CANMV_V2) || defined(CONFIG_BOARD_K230_CANMV_01STUDIO)
+#if defined(CONFIG_BOARD_K230_CANMV)
+    k_connector_type connector_type = NT35516_MIPI_2LAN_540X960_30FPS;
+#elif defined(CONFIG_BOARD_K230_CANMV_V2) || defined(CONFIG_BOARD_K230_CANMV_01STUDIO)
 	k_connector_type connector_type = LT9611_MIPI_4LAN_1920X1080_30FPS;// HX8377_V2_MIPI_4LAN_1080X1920_30FPS;
 #elif defined(CONFIG_BOARD_K230D_CANMV) || defined(CONFIG_BOARD_K230D_CANMV_BPI) || defined(CONFIG_BOARD_K230_CANMV_LCKFB)
     k_connector_type connector_type = ST7701_V1_MIPI_2LAN_480X800_30FPS;
 #elif defined(CONFIG_BOARD_K230_CANMV_DONGSHANPI)
@@ -334,7 +348,7 @@ static k_s32 vo_layer_vdss_bind_vo_config(void)
 
     sample_connector_init();
 
-#if defined(CONFIG_BOARD_K230D_CANMV) || defined(CONFIG_BOARD_K230_CANMV_DONGSHANPI) || defined(CONFIG_BOARD_K230D_CANMV_BPI) || defined(CONFIG_BOARD_K230_CANMV_LCKFB)
+#if defined(CONFIG_BOARD_K230_CANMV) || defined(CONFIG_BOARD_K230D_CANMV) || defined(CONFIG_BOARD_K230_CANMV_DONGSHANPI) || defined(CONFIG_BOARD_K230D_CANMV_BPI) || defined(CONFIG_BOARD_K230_CANMV_LCKFB)
     info.act_size.width = ISP_CHN0_HEIGHT;//1080;//640;//1080;
     info.act_size.height = ISP_CHN0_WIDTH;//1920;//480;//1920;
     info.format = PIXEL_FORMAT_YVU_PLANAR_420;
@@ -449,11 +463,11 @@ int sample_vivcap_init( void )
     k_s32 ret = 0;
 
 #if defined(CONFIG_BOARD_K230_CANMV)
-    sensor_type = OV_OV5647_MIPI_CSI0_1920X1080_30FPS_10BIT_LINEAR;
+    sensor_type = GC2093_MIPI_CSI2_1920X1080_30FPS_10BIT_LINEAR;
 #elif defined(CONFIG_BOARD_K230_CANMV_V2) || defined(CONFIG_BOARD_K230_CANMV_01STUDIO)
     sensor_type = OV_OV5647_MIPI_CSI2_1920X1080_30FPS_10BIT_LINEAR_V2;
 #elif defined(CONFIG_BOARD_K230_CANMV_DONGSHANPI)
-    sensor_type = GC2093_MIPI_CSI2_1920X1080_30FPS_10BIT_LINEAR;
+    sensor_type = OV_OV5647_MIPI_CSI2_1920X1080_30FPS_10BIT_LINEAR_V2;
 #elif defined(CONFIG_BOARD_K230D_CANMV) || defined(CONFIG_BOARD_K230D_CANMV_BPI)
     sensor_type = OV_OV5647_MIPI_1920X1080_30FPS_10BIT_LINEAR;
 #elif defined(CONFIG_BOARD_K230_CANMV_LCKFB)
